### ISSUE TITLE
Can't integrate with ApolloServer via `modules`

### ISSUE THREAD

**Issue by dancon** [Urigo/graphql-modules#427]
```
// or use `modules` in `ApolloServer`
const server = new ApolloServer({
  modules: [
    AppModule
  ],
  context: session => session,
});
```

I try to user code above to integrate with ApolloServer, but compiled by ts return a error like below:

```
TSError: ⨯ Unable to compile TypeScript:
src/index.ts:8:3 - error TS2322: Type 'GraphQLModule<any, any, any>[]' is not assignable to type 'GraphQLSchemaModule[]'.
  Type 'GraphQLModule<any, any, any>' is not assignable to type 'GraphQLSchemaModule'.
    Types of property 'resolvers' are incompatible.
      Type 'IResolvers<any, any>' is not assignable to type 'GraphQLResolverMap<any>'.
        Index signatures are incompatible.
          Type '(() => any) | GraphQLScalarType | IEnumResolver | IResolverObject<any, any, any> | IResolverOptions<any, any, any>' is not assignable to type '{ [fieldName: string]: GraphQLFieldResolver<any, any, { [argName: string]: any; }> | { requires?: string | undefined; resolve: GraphQLFieldResolver<any, any, { [argName: string]: any; }>; }; }'.
            Type '() => any' is not assignable to type '{ [fieldName: string]: GraphQLFieldResolver<any, any, { [argName: string]: any; }> | { requires?: string | undefined; resolve: GraphQLFieldResolver<any, any, { [argName: string]: any; }>; }; }'.
              Index signature is missing in type '() => any'.

8   modules: [ userModule ],
    ~~~~~~~

  node_modules/apollo-server-core/dist/types.d.ts:27:5
    27     modules?: GraphQLSchemaModule[];
           ~~~~~~~
    The expected type comes from property 'modules' which is declared here on type 'ApolloServerExpressConfig & { cors?: boolean | CorsOptions | undefined; }'
```

**Comment by eiddie** [comment]
`AppModule` should be `GraphQLSchemaModule` instead of  `GraphQLModule`

**Comment by ardatan** [comment]
@dancon I think the problem is related to TypeScript typings, so you can use `any` for workaround until we fix it;
```
const server = new ApolloServer({
  modules: [
    AppModule as any
  ],
  context: session => session,
});
```

**Comment by dancon** [comment]
@ardatan 👌

**Comment by ardatan** [comment]
Fixed in 0.7.6!
